dist/stat_data.py

```diff
@@ -507,6 +457,80 @@ def __init__(self, name, desc, flags=''):
     DhandleStat('dh_sweep_tod', 'connection sweep time-of-death sets'),
     DhandleStat('dh_sweeps', 'connection sweeps'),
 
+    ##########################################
+    # Eviction server statistics
+    ##########################################
+    EvictionServerStat('cache_eviction_internal_pages_already_queued', 'internal pages seen by eviction walk that are already queued'),
```

Comment:
Should this now starts with eviction instead of cache_eviction?

Reply:
I agree with you about renaming them into `eviction` by removing the `cache` prefix. Currently, I want to limit the changes as part of this PR. I will create another ticket to handle the renaming of the stats.